src/components/Post/Comment/CommentsContainer/CommentCard.tsx

```
@@ -0,0 +1,174 @@
+	const { asPath } = useRouter();
+	const commentScrollRef = useRef<HTMLDivElement>(null);
+	const [newSentiment, setNewSentiment] = useState<number>(sentiment || 0);
+	const {
+		postData: { postIndex, postType }
+	} = usePostDataContext();
+	const [openModal, setOpenModal] = useState<boolean>(false);
+	useEffect(() => {
+		if (typeof window == 'undefined') return;
+		const hashArr = asPath.split('#');
+		const hash = hashArr[hashArr.length - 1];
+		if (commentScrollRef && commentScrollRef.current && hash === `${id}`) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)